src/main/resources/application-local.yml

```diff
@@ -12,12 +12,19 @@ powsybl:
       
 gridsuite:
   services:
-    study-server:
+    -
```

Comment:
It was not working anymore with old version of this section ?

Reply:
Why ?

Reply:
so we can get the map of services and get the baseUri of the server that we need to call based on parameters type